src/server.py

```diff
@@ -416,7 +414,8 @@ def run_seta_details_query():
     date = request.args.get("date", "")
     active = request.args.get("active", 0)
     buildbot = request.args.get("buildbot", 0)
-    branch = request.args.get("branch", '')
```

Comment:
hmm, because I think the branch in here is about to generate a buildername. e.g mozilla-inbound for Ubuntu VM 12.04 x64 **mozilla-inbound** opt test mochitest-devtools-chrome-3. But we could get it from jobnames endpoint directly. (Actually, we don't have too much choice for now, we only get mozilla-inbound data from runnable job endpoint.